modules/reportgen.py

```
@@ -1,62 +1,96 @@
 def get_vulnerable_versions(vulnerability):
     versions_text = ""
-    cve_org_data = vulnerability['namespaces'].get('cve.org', {})
-    vendor_data = cve_org_data.get('affects', {}).get('vendor', {}).get('vendor_data', [])
-    for vendor in vendor_data:
-        product_data = vendor.get('product', {}).get('product_data', [])
-        for product in product_data:
-            if 'version' in product:
-                version_data = product['version'].get('version_data', [])
-                vulnerable_versions = [version['version_value'] for version in version_data]
-                versions_text += "\n".join([f"- {version}" for version in vulnerable_versions])
+    cve_org_data = vulnerability['namespaces'].get('nvd_nist_gov', {}).get('cve', {})
```

Ruff

Local variable `cve_org_data` is assigned to but never used

Remove assignment to unused variable `cve_org_data`

(F841)